### PR TITLE
Update author blurb and articles

### PR DESCRIPTION
**Pull request by iranzo**
## Prerequisites

- [x] My Pull Request is filled against the `next` branch
- [ ] I have updated the [CHANGELOG](https://github.com/Pelican-Elegant/elegant/blob/next/CHANGELOG.md)

## Recommended Steps

- [ ] My patch adds a new feature, therefore I have also added a help article about it
- [ ] My patch changes Elegant behavior, therefore I have updated the help article to reflect this change
- [x] My commits are [signed](https://help.github.com/en/articles/signing-commits)

### CODE REVIEW

File: documentation/pelicanconf.py [hunk 1]
```diff
@@ -8,6 +8,14 @@
 SITESUBTITLE = "The Best Pelican Theme"
 SITEURL = ""
 
+AUTHORS = {
```
Comment (talha131) [inline documentation/pelicanconf.py]: @iranzo you should add your entry to the existing dictionary that has my entry in it too.

**Comment by iranzo** [comment]
That's why this is still a WIP, just saved what I had before it's final,
don't worry about it 'yet'
-- 

Pablo Iranzo Gómez

GnuPG: 0x5BD8E1E4
Web: https://iranzo.github.io  ||  https://citellus.org

File: documentation/pelicanconf.py (hunk 1 above)
Comment (iranzo) [reply documentation/pelicanconf.py]: yup, done in next push

**Comment by iranzo** [comment]
BTW, I did it using 'git blame' on the files to find my name, probably others can do the same

**Comment by talha131** [comment]
> BTW, I did it using 'git blame' on the files to find my name, probably others can do the same

Cool tip.

**Comment by talha131** [comment]
@all-contributors please add @iranzo for question, bug, blog, code, content, doc, ideas, maintenance, projectManagement, review, test, tutorial

**Comment by talha131** [comment]
> That's why this is still a WIP, just saved what I had before it's final, don't worry about it 'yet'

M bad. Some how I missed that it's a draft PR.

**Pull request merged**